### PR TITLE
Exclude props

### PR DESCRIPTION
**Pull request by istarkov**
## Description

ref https://github.com/webstudio-is/webstudio-builder/issues/2151

Exclude following props from radix components

```
-e asChild -e modal -e defaultValue -e defaultOpen
```

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/generate-arg-types/src/cli.ts [hunk 1]
```diff
@@ -14,7 +15,24 @@ const options = {
...
+      multiple: true,
+      short: "e",
+    },
+  },
+  strict: true,
+} as const satisfies ParseArgsConfig;
+
+const cliArgs = parseArgs({ args: process.argv.slice(2), ...CLI_ARGS_OPTIONS });
+
+console.log(cliArgs);
```
Comment (TrySound) [inline packages/generate-arg-types/src/cli.ts:32]: Remove console?

File: packages/generate-arg-types/src/cli.ts [hunk 2]
```diff
@@ -14,7 +15,24 @@ const options = {
...
+
+const componentsGlobString = cliArgs.positionals.join(" ");
```
Comment (TrySound) [inline packages/generate-arg-types/src/cli.ts:34]: `const globs = cliArgs.positionals`?

**Review by TrySound — APPROVED** [review]
Please remove explicit Omit in our components

**Comment by kof** [comment]
why is this not done over type Omit like we did before?

**Comment by istarkov** [comment]
Same props, same names, done only for radix lib. There is no need to wrap every component and omit there.

File: packages/generate-arg-types/src/cli.ts (hunk 2 above)
Comment (istarkov) [reply packages/generate-arg-types/src/cli.ts:34]: They are inside '' to not allow system expand, so same logic as before

**Comment by istarkov** [comment]
With Omit is so easy to forget to remove them at all.

File: packages/generate-arg-types/src/cli.ts (hunk 2 above)
Comment (TrySound) [reply packages/generate-arg-types/src/cli.ts:32]: Right after this there is `componentsGlobString.split(' ')`. What system expand?

Comment (istarkov) [reply packages/generate-arg-types/src/cli.ts:32]: system expands pathes not inside ''

Comment (istarkov) [reply packages/generate-arg-types/src/cli.ts:32]: `echo */*` is not same as `echo '*/*'`

Comment (istarkov) [reply packages/generate-arg-types/src/cli.ts:32]: https://github.com/webstudio-is/webstudio-builder/blob/ab903af1303180f178a6d464709742d7381539e6/packages/sdk-components-react-radix/package.json#L44

Comment (TrySound) [reply packages/generate-arg-types/src/cli.ts:32]: Got it. Could be a one liner with comment `const globs = cliArgs.positionals.join(" ").split(" ")`

**Pull request merged**